diagnostic.md

```diff
@@ -35,7 +36,13 @@ Record your responses inside the fenced code blocks below each question.
     ```
 
     ```md
-    <!-- your response here -->
+    The first route definition is nested, so all references to it must reference
+    its parent path, and content from the child route will be displayed simul-
+    taneously with content from the parent route.
+
+    The second defintion is not nested, so it can be referenced without reference
+    to its parent, and it will not be displayed simultaneously with the parent
+    route's data
```

Comment:
These two route definitions are identical  URL paths.  
Nested route definitions, do require a nested file structure.  
One of the biggest distinctions is where Ember will render the content.  
The nested `'products'/'product'` instructs Ember to render
`app/products/product/template.hbs` in the `outlet` of the parent `app/products/template.hbs`.  
The non-nested `'product'` instructs Ember to render `app/product/template.hbs` in the context of the application template.